examples/datadog-agent-logs-apm.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: datadoghq.com/v1alpha1
+kind: DatadogAgent
+metadata:
+  name: datadog
+spec:
+  credentials:
+    apiKey: <DATADOG_API_KEY>
+    appKey: <DATADOG_APP_KEY>
+  agent:
+    image:
+      name: "datadog/agent:latest"
+    config:
+      logLevel: "DEBUG"
+      leaderElection: true
```

Comment:
this is possible but I don't think we should advertise it anymore (making the node agents elect a leader), we don't need that anymore with the cluster agent